asset/css/page-nav.css

```diff
@@ -35,10 +35,11 @@
 
 /* Responsive site navigation */
 
-/* Hides page navigation on screen widths smaller than 1300px */
-@media screen and (max-width: 1299.98px) {
+/* Hides page navigation on screen widths smaller than 1300px, or when printing */
+@media screen and (max-width: 1299.98px), print {
```

Comment:
Although this implementation is convenient and works, we should separate the sections (responsive design and printing). This also keeps consistency with the implementation in `site-nav.css`.

You should use the same `display: none` style instead of `width: 0` for `@media print` as well.